File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1009.cpp
#include <stdio.h>

int main(){
    char nome[30];
    double salario = 0;
    double totalVendas = 0;
    double salarioTotal = 0;
    double comissao = 0.15;
    
    scanf("%s", nome);
    scanf("%lf", &salario);
    scanf("%lf", &totalVendas);
    
    salarioTotal = (totalVendas * comissao) + salario;
    
    printf("TOTAL = R$ %.2lf\n", salarioTotal);
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python/Iniciante/1009.py
nome = raw_input() #ler valores
salario = input()
vendas = input()

comissao = vendas * 0.15 #15% das vendas
salarioTotal = comissao + salario #soma salario com comissao

print("TOTAL = R$ %.2f" % salarioTotal) #imprimi salario final